spec.md

```diff
@@ -74,7 +74,7 @@ The `Span` interface must have the following capabilities:
 - Create and start a new child `Span` with a given operation name. Any trace attributes must be passed through into the child. **(py: `start_child`, go: `StartChild`)**
 - Finish the (already-started) `Span`.  Finish should be the last call made to any span instance, and to do otherwise leads to undefined behavior. **(py: `finish`, go: `Finish`)**
 - Set a key:value tag on the `Span`. The key must be a `string`, and the value must be either a `string`, a `boolean`, or a numeric type. Behavior for other value types is undefined. If multiple values are set to the same key (i.e., in multiple calls), implementation behavior is also undefined. **(py: `set_tag`, go: `SetTag`)**
-- Add a new log event to the `Span`, accepting an event name `string` and an optional structured payload argument. If specified, the payload argument may be of any type and arbitrary size, though implementations are not required to retain all payload arguments (or even all parts of all payload arguments). **(py: `log_event, log_event_with_payload`, go: `LogEvent, LogEventWithPayload`)**
+- Add a new log event to the `Span`, accepting an event name `string` and an optional structured payload argument. If specified, the payload argument may be of any type and arbitrary size, though implementations are not required to retain all payload arguments (or even all parts of all payload arguments). An optional timestamp can be used to specify a past timestamp. **(py: `log`, go: `Log`)**
```

Comment:
@michaelsembwever there is a reasonable case to be made for consolidation around `log` in python (and indeed the examples in `use-cases.md` are pythonic)... as @yurishkuro points out, Go does not make this pretty or easy... We do presently have a `Log(data LogData)` method, but that's just the plumbing API (as you would put it!).

So if you want to point to `log`/`Log` here, that's ok with me. But if you actually want to remove all other signatures from the various implementations, I'd consider that a regression for Go.